frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/raw-capacity-card/raw-capacity-card.tsx

```diff
@@ -39,6 +46,21 @@ const RawCapacityCard: React.FC = React.memo(() => {
   const usedCapacityAdjusted = humanizeBinaryBytes(usedCapacityMetric);
   const availableCapacityAdjusted = humanizeBinaryBytes(totalCapacityMetric - usedCapacityMetric);
 
+  const capacityRatio = usedCapacity.value / totalCapacity.value;
```

Comment:
```suggestion
  const capacityRatio = parseFloat((usedCapacity.value / totalCapacity.value).toFixed(2));
```
Should avoid re-renders for very small changes in storage usage.